server/src/main/java/org/apache/cloudstack/vm/VmImportManagerImpl.java

```diff
@@ -233,10 +233,10 @@ private UnmanagedInstanceResponse createUnmanagedInstanceResponse(UnmanagedInsta
         response.setMemory(instance.getMemory());
         response.setOperatingSystemId(instance.getOperatingSystemId());
         response.setOperatingSystem(instance.getOperatingSystem());
-        response.setObjectName(UnmanagedInstance.class.getSimpleName().toLowerCase());
+        response.setObjectName("unmanagedinstance");
```

Comment:
? why this change? I am confused both about the initial code and the changed code. Seems not important, but still I am interested in naming per sé. What are the considerations?

Reply:
@DaanHoogland Because earlier the class name was `UnmanagedInstance` so it could be easily used as object name for API response but since after review I changed the class name to `UnmanagedInstanceTO` I had to explicitly mention object name